docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md

```diff
@@ -0,0 +1,73 @@
+| [nebula-java](https://github.com/vesoft-inc/nebula-java)               | The official Python client of Nebula Graph             | [Graph Client Example](https://github.com/vesoft-inc/nebula-java#graph-client-example)                                                                    |
+
+If you don't have a Nebula Graph database yet, we recommend that you try the cloud service. [Nebula Graph Cloud Service](https://www.nebula-cloud.io/) supports on-demand deployment and fast building, and uses Nebula Graph Studio as its default client.
+
+## Use nebula-console to connect to Nebula Graph
+
+1. Download [the nebula-console binary file](TODO) [Coming soon].
+
+    ```bash
+    $ wget [Coming soon]
+    ```
+
+2. Use the binary file to connect to Nebula Graph.
+
+    ```bash
+    $ ./nebula-console -addr <ip> -port <port> -u <username> -p <password> 
+    [-t 120] [-e "nGQL_statement" | -f filename.nGQL]
+    ```
+
+    | Option       | Description                                                                                                                                                                   |
+    | ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | -h           | Show the help menu.                                                                                                                                                           |
+    | -addr        | Set the IP address of the graphd service. The default address is 127.0.0.1.                                                                                                   |
+    | -port        | Set the port number of the graphd service. The default port number is 3699.                                                                                                   |
+    | -u/-user     | Set the username of your Nebula Graph account. The default username is root.                                                                                                  |
+    | -p/-password | Set the password of your Nebula Graph account. The default password for the root user is nebula.                                                                              |
+    | -t/-timeout  | Set an integer-type timeout threshold of the connection. The unit is second. The default value is 120.                                                                        |
+    | -e/-eval     | Set a string-type nGQL statement. The nGQL statement is executed once the connection succeeds. The connection stops after the result is returned..             |
+    | -f/-file     | Set the path of an nGQL file. The nGQL statements in the file are executed once the connection succeeds. You'll get the return messages and the connection stops then. |
```

Comment:
all the option name must be in code format

in the Description column, use third-person singular verbs to start the sentences.

Reply:
It is not necessary to use the code format for table headers, whether they are options or parameters. But it is good to have a standard style.